doc/source/3dgraphics/glgraphitem.rst

```diff
@@ -0,0 +1,7 @@
+GLGraphItem
+===========
+
+.. autoclass:: pyqtgraph.opengl.GLGraphItem
+	:members:
+
+	.. automethod:: pyqtgraph.opengl.GLGridItem.__init__
```

Comment:
Still have `GridItem` here. I would actually remove this line since `__init__` doesn't have its own docstring. Typically docs for `__init__` go at the top level but pyqtgraph has a convention of splitting it into a) class description and b) `__init__` args.

Reply:
And come to think of it, it might be good to state in the top-level docstring that keyword arguments are passed to `setData`.

Reply:
Oof..that's embarrassing about GLGridItem, nice catch... and yeah, good call on putting the required args.